src/averageAges.js

```diff
@@ -14,12 +14,15 @@
  *
  * @return {number}
  */
+
+const getAvarageAge = people => people
+  .reduce((acc, { born, died }) => acc + (died - born), 0) / people.length;
```

Comment:
The 'getAverageAge' function could return NaN if the 'people' array is empty. You should add a check to see if 'people.length' is not zero before performing the division.